data/102/031/611/102031611.geojson

```diff
@@ -66,13 +66,14 @@
         "\u039a\u03cc\u03bf\u03c5\u03c4\u03c3\u03b9"
     ],
     "name:eng_x_preferred":[
-        "Kochi-shi"
+        "K\u014dchi"
```

Comment:
`Kochi` and `wof:name`

maybe you could look for the `\u` escape in the text generally to avoid this in the future?